tests-extension/Makefile

```
@@ -0,0 +1,24 @@
+	@echo "Building extension binary..."
+	@cd test/e2e && $(MAKE) -f bindata.mk update-bindata
+	@mkdir -p bin
+	CGO_ENABLED=0 GO_COMPLIANCE_POLICY="exempt_all" GOTOOLCHAIN=auto go build -mod=vendor -o $(BINARY) ./cmd
+	@echo "Binary built: $(BINARY)"
+
+.PHONY: clean
+clean:
+	@rm -f $(BINARY)
+	@cd test/e2e && $(MAKE) -f bindata.mk clean-bindata
+
+.PHONY: help
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)